### PR TITLE
fix: fix decimal regex pattern

### PR DESCRIPTION
**Pull request by swetshaw**
The current decimal regex pattern was able to accept only two-digit values. Changed it to take one-digit values as well.

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -221,7 +221,7 @@ components:
           $ref: "#/components/schemas/Address"
         decimal: 
           type: string
-          pattern: '^[1-9][0-9]$'
+          pattern: '^[1-9]?\d{1}$'
```
Comment (adiwajshing) [inline openapi.yaml]: A better pattern may be: `^[1-9]{1,3}$` to allow for 1 to 3 digit numbers

Comment (swetshaw) [reply openapi.yaml]: `^[1-9]{1,3}$` this won't include 0 in ones and tens place,  how about this - `^[1-9][0-9]{0,2}` ? @adiwajshing

Comment (adiwajshing) [reply openapi.yaml]: Yep, works

**Comment by swetshaw** [comment]
Hey @adiwajshing , please approve for merge

**Review by adiwajshing — APPROVED** [review]

**Pull request merged**